.github/workflows/flakeguard.yml

```diff
@@ -322,9 +322,11 @@ jobs:
         id: gen_id
         run: echo "id=$(uuidgen)" >> "$GITHUB_OUTPUT"
 
-      - name: Install flakeguard
+      - name: Install flakeguard and gotestsum
         shell: bash
-        run: go install github.com/smartcontractkit/chainlink-testing-framework/tools/flakeguard@bc66d27e3325a1fecedc0fc932d13d3e54ff6f25 # flakguard@0.1.0
+        run: |
+          go install github.com/smartcontractkit/chainlink-testing-framework/tools/flakeguard@bc66d27e3325a1fecedc0fc932d13d3e54ff6f25 # flakguard@0.1.0
+          go install gotest.tools/gotestsum@latest # needed for flakeguard output formatting
```

Comment:
Should we pin this to a specific version?

Reply:
Pinned it everywhere: 1ce56f4dd6e412373726a42dd701c6dde4b08f71